Source/Filter Effects/feBlend/SvgBlendMode.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.ComponentModel;
+
+namespace Svg.FilterEffects
+{
+    [TypeConverter(typeof(EnumBaseConverter<SvgBlendMode>))]
+    public enum SvgBlendMode
+    {
+        Normal,
+        Multiply,
+        Screen,
+        Darken,
+        Lighten
+    }
+}
```

Comment:
This is from SVG 1.1, not sure if we want to add the SVG 2 additions (I has in [my PR](https://github.com/vvvv/SVG/pull/497/files#diff-6d83bf45faed77dbf6526d6089f96f76)).

Reply:
> This is from SVG 1.1, not sure if we want to add the SVG 2 additions (I has in [my PR](https://github.com/vvvv/SVG/pull/497/files#diff-6d83bf45faed77dbf6526d6089f96f76)).

I can support those is my Svg.Skia to I will add them in PR, may be useful in feature for others.